go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/evergreen-ci/aviation
 
-go 1.16
+go 1.20
```

Comment:
Made this go 1.20 because that's what Evergreen is on and we need at least 1.18 to take advantage of the bug fix.